h2o-py/h2o/h2o.py

```diff
@@ -759,13 +766,18 @@ def parse_setup(raw_frames, destination_frame=None, header=0, separator=None, co
     assert_is_type(column_types, [coltype], {str: coltype}, None)
     assert_is_type(na_strings, [natype], {str: natype}, None)
     assert_is_type(partition_by, None, [str], str)
+    assert_is_type(quotechar, str, None)
```

Comment:
```suggestion
    assert_is_type(quotechar, None, U("'", '"'))
```

Reply:
You probably mean `assert_is_type(quotechar,None, U("'", '"'))`

Reply:
used